Sparkle_custom.gyp

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2011 House of Life Property ltd.
+# Copyright (c) 2011 Crystalnix <vgachkaylo@crystalnix.com>
+
+{
+  'conditions': [
+    ['OS=="mac"', {
+      'targets': [
+        {
+          'target_name': 'Sparkle',
+          'type': 'shared_library',
+          'product_name': 'Sparkle',
+          'actions': [
+            {
+              'action_name': 'sparkle_make_release',
+              'inputs': [
+                'Makefile',
+                '<!@(ls -1 Sparkle/*.h)',
+                '<!@(ls -1 Sparkle/*.m)',
+              ],
+              'outputs': [
+                'Build/Products/Release/Sparkle.framework'
+              ],
+              'action': [
+                # this is the command that `make release` runs,
+                # except this doesn't open a finder window when it is done
+                '/usr/bin/env', 'bash', '-c',
```

Comment:
yes can you cut 3rd party copyrights if unneeded? is /usr/bin/env ok for jenkins? mavericks?